docs/user_manual/calculations.md

```diff
@@ -409,6 +410,16 @@ crash due to an unobservable system, but succeeding with the calculations and gi
 
 Algorithm call: `CalculationMethod.iterative_linear`
 
+### Short circuit algorithms
+
+**TODO**
+
+```{note}
+In case of short circuit calculations, the zero-sequence parameters are required only
+if any of the faults in any of the scenarios within a batch are not three-phase faults
+(i.e. `fault_type` is not `FaultType.three_phase`).
+```
```

Comment:
```suggestion
```{note}
In case of short circuit calculations, the zero-sequence parameters are required only
if any of the faults in any of the scenarios within a batch are not three-phase faults
(i.e. `fault_type` is not `FaultType.three_phase`).
```
```
Agreed with the part that this is not where the user would try to find the info.